 jni: MediaPlayer: Improve the handling of lib loading

 - Only try to load the extmedia lib once.
 - Also adjust some logging levels.

diff --git a/media/jni/android_media_MediaPlayer.cpp b/media/jni/android_media_MediaPlayer.cpp
index 71550be..7320f07 100644
--- a/media/jni/android_media_MediaPlayer.cpp
+++ b/media/jni/android_media_MediaPlayer.cpp
@@ -182,17 +182,23 @@ CheckExtMediaFn JNIMediaPlayerFactory::sExtMediaFn =
 CreateNativeQCMediaPlayerFn JNIMediaPlayerFactory::sNativeQCMediaPlayerFn =
         JNIMediaPlayerFactory::loadNativeQCMediaPlayer();
 
+bool load_fail = false;
 
 void JNIMediaPlayerFactory::loadLib()
 {
+    if (load_fail) {
+        return;
+    }
+
     if (!mLibHandle) {
         mLibHandle = ::dlopen(EXTMEDIAJNI_LIB_NAME, RTLD_LAZY);
         if (!mLibHandle) {
-            ALOGV("%s", dlerror());
+            ALOGI("%s", dlerror());
+            load_fail = true;
             return;
         }
-    ALOGV("Opened %s", EXTMEDIAJNI_LIB_NAME);
-  }
+    ALOGI("Opened %s", EXTMEDIAJNI_LIB_NAME);
+    }
 }
 
 CreateJNIExtMediaPlayerListenerFn JNIMediaPlayerFactory::loadJNIExtMediaPlayerListener()
@@ -267,7 +273,7 @@ bool JNIMediaPlayerFactory::checkExtMedia(JNIEnv *env, jobject thiz)
     if (sExtMediaFn) {
         bIsQCMediaPlayerPresent = (*sExtMediaFn)(env, thiz);
     }
-    ALOGE("JNIMediaPlayerFactory: bIsQCMediaPlayerPresent %d", bIsQCMediaPlayerPresent);
+    ALOGI("JNIMediaPlayerFactory: bIsQCMediaPlayerPresent %d", bIsQCMediaPlayerPresent);
     return bIsQCMediaPlayerPresent;
 }
 
